lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
```

Comment:
fwiw, checking `isNaN(value)` should also work here since `Date.prototype.valueOf()` will return `NaN` when the date is invalid.